package.json

```diff
@@ -17,14 +17,12 @@
     "node": ">=8.10.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "~11.0.0",
     "@commitlint/config-conventional": "~11.0.0",
```

Comment:
@macintoshpie  Should this be removed, too?

Reply:
ope yeah, lemme do that